### ISSUE TITLE
Enhance RANCID parser to support optional 4th field in router.db files

### ISSUE THREAD

**Issue by jathanism** [trigger/trigger#202]
See: http://www.shrubbery.net/rancid/man/router.db.5.html.

The format `<device_name>:<device_type>:<state>[:comments]` (with comments optional) is valid.

Trigger's RANCID NetDevices parser should be enhanced to support this.

**Comment by coxley** [comment]
Since the CSV loader also uses the Rancid parser, this would prevent CSV files from providing device platform.

Would either need to separate CSV parser or (easier and most likely) just use a condition for what the delimeter is.

**Comment by jathanism** [comment]
Yes, you are correct. I think there's already that condition, but maybe I'm wrong?

Ref: https://github.com/trigger/trigger/blob/develop/trigger/netdevices/loaders/filesystem.py#L153

**Comment by coxley** [comment]
It would be trivial to do so, but currently there's no condition for the parsing. _parse_delimited_file() cares not what the delimeter actually is.

Ref: https://github.com/trigger/trigger/blob/develop/trigger/rancid.py#L56

**Comment by jathanism** [comment]
Ohhhh yeah. I remember that now. Ugh.
